examples/integrate-with-react/package.json

```diff
@@ -19,5 +19,11 @@
     "test:watch": "react-scripts test --watchAll=false",
     "eject": "react-scripts eject",
     "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache"
+  },
+  "jest": {
+    "moduleNameMapper": {
+      "^nimma/legacy$": "<rootDir>/../../node_modules/nimma/dist/legacy/cjs/index.js",
+      "^nimma/(.*)": "<rootDir>/../../node_modules/nimma/dist/cjs/$1"
+    }
```

Comment:
Whats these for? 🤔

Reply:
https://github.com/stoplightio/spectral/issues/2002#issuecomment-994770572 :)

Reply:
Do everyone who integrates the parser, now have to do this if they use Jest? 🤔

Reply:
Yep, I described that in this PR https://github.com/asyncapi/parser-js/pull/645 in the Readme.